libs/template/presets/fulfillment/app.ts

```diff
@@ -24,6 +27,16 @@ export function fulfillmentFeatures(
     uiFeature,
     cartFeature,
     coreFeature,
+    siteFeature,
+    formFeature,
+    {
+      //drop PageMetaResolver from experienceFeature
+      //to exclude unnecessary functionality from SPA
+      providers: experienceFeature.providers?.filter(
+        (feature) => ![PageMetaResolver].includes(feature.provide)
+      ),
+      components: experienceFeature.components,
+    },
```

Comment:
Is it still relevant?

Reply:
Due to FA still not use static experience data for its pages (components instead) and PageMetaResolver is not satisfied with this

Reply:
also we'll use it shortly for the work Alexander is doing